libraries/rush-lib/src/logic/operations/OperationResultSummarizerPlugin.ts

```diff
@@ -253,6 +257,14 @@ function writeDetailedSummary(
       terminal.write(details);
     }
 
+    if (shouldPrintLogFilePaths) {
+      if (operationResult.status === OperationStatus.Failure && operationResult.logFilePaths !== undefined) {
+        terminal.writeLine('');
+        terminal.writeLine(Colorize.gray(`stdout: ${operationResult.logFilePaths.text}`));
```

Comment:
Note that this log contains both stdout and stderr.

Reply:
Oh, I didn't know that.

In such case, I guess single line `log file: xyz` should be enough.

Reply:
![image](https://github.com/user-attachments/assets/58c9bab2-b9d3-4285-b880-655db60c2542)

Done